src/nyc_trees/apps/core/models.py

```diff
@@ -88,6 +95,10 @@ def clean(self):
     def get_absolute_url(self):
         return reverse('user_detail', args=[self.username])
 
+    def completed_training(self):
```

Comment:
I just realized that this kind of shadows the existing ``online_training_complete`` boolean flag. I think we may reduce possible confusion by moving the calculation of whether or not training is complete out to the view.